### ISSUE TITLE
✅ ♿ 🚀 Feature Request: Elementor Pro > Carousel add Alt for image (solution is ready)

### ISSUE THREAD

**Issue by simon-zk** [elementor/elementor#14052]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
The carousel sometimes needs an image title or alt, even though their output is not as img, but rather a background, image alt or title can be used with JS.

**Describe the solution you'd like**
On file: elementor-pro/modules/carousel/widgets/media-carousel.php
Line ±272, containing:
`'class' => 'elementor-carousel-image',`
Add this
`'alt' => get_post_meta(($slide['image']['id']), '_wp_attachment_image_alt', TRUE),`

It will output the alt of every image in the div as it outputs the class and the background image style.
It's tested and used. I hate to update it after every the elementor update. I think it will be useful for someone, plus for additional features in future, such as availability to so simply select that you want to show the image title above the carousel.

**Describe alternatives you've considered**
This is the best solution. If not you should be making additional requests after it's loaded, to get alt for each image.

**Additional context**
I hope this will be updated as soon as possible :))

**Comment by shilo-ey** [comment]
Don't you think `aria-label` is preferred?
https://stackoverflow.com/a/31086818

Thanks

**Comment by simonkane** [comment]
It can be `aria-label`, there's really no difference.

**Comment by shilo-ey** [comment]
There is because semantically you can't add `alt` to `div`s

**Comment by simonkane** [comment]
You can alt any attribute to divs and my provided example is just showing that it would be helpful having the image Title or Alt inside the div, no matter how it's inserted there, as long as doesn't break the code and can be used with javascript.

**Comment by shilo-ey** [comment]
But the solution is not to allow only a fix to your case but to all of our users, with minimum markup changes.

Anyway, we will look into this.

Thanks for raising this issue

**Comment by TheRashad** [comment]
Hey, is there any quick solution for elementor pro slides?

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro **v3.13.0**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
